### PR TITLE
Update tensorflow branch

### PR DESCRIPTION
**Pull request by fprost**
Contains:
- Adaptation of run.py for tf model
- Change of attention layers
- A new shell script

### CODE REVIEW

File: experiments/tf_trainer/tf_gru_attention/run.py [hunk 1]
```diff
@@ -22,15 +22,22 @@
 tf.app.flags.DEFINE_string("embeddings_path",
                            "local_data/glove.6B/glove.6B.100d.txt",
                            "Path to the embeddings file.")
+tf.app.flags.DEFINE_boolean("is_binary_embedding", True,
```
Comment (nthain) [inline experiments/tf_trainer/tf_gru_attention/run.py]: We should probably have the default flag match the default embedding path. So, that would make it False, right?

File: experiments/tf_trainer/tf_gru_attention/run.py [hunk 2]
```diff
@@ -44,27 +51,40 @@ def main(argv):
...
   dataset = tfrecord_input.TFRecordInput(
       train_path=FLAGS.train_path,
       validate_path=FLAGS.validate_path,
       text_feature=text_feature_name,
       labels=LABELS,
-      feature_preprocessor=tokenize_op,
+      feature_preprocessor_init=tokenize_op_init,
       batch_size=FLAGS.batch_size)
 
+  # TODO: Move embedding *into* Keras model.
```
Comment (nthain) [inline experiments/tf_trainer/tf_gru_attention/run.py]: What does this mean?

File: experiments/tf_trainer/tf_gru_attention/run.py (hunk 1 above)
Comment (fprost) [reply experiments/tf_trainer/tf_gru_attention/run.py]: Yes good point. Will change

File: experiments/tf_trainer/tf_gru_attention/run.py (hunk 2 above)
Comment (fprost) [reply experiments/tf_trainer/tf_gru_attention/run.py]: This comment was written by someone else.

I am assuming that the goal here would be to have the embeddings directly in the model function (keras and tf model.py) and therefore get rid of this function add_embedding_to_model.

**Pull request merged**